File: Java_Course/src/App/BetterCalculator.java
package App;

import java.util.Scanner;

public class BetterCalculator {
	public static void Calculator(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		System.out.print("Enter a number: ");
		double num1 = keyboardInput.nextDouble();
		
		System.out.print("Enter an operator: ");
		String op = keyboardInput.next();
		
		System.out.print("Enter the second number: ");
		double num2 = keyboardInput.nextDouble();
		
		if(op.equals("+")) {
			System.out.println( num1 + num2 );
		}
		else if(op.equals("-")) {
			System.out.println( num1 - num2 );
		}
		else if(op.equals("*")) {
			System.out.println( num1 * num2 );
		}
		else if(op.equals("/")) {
			System.out.println( num1 / num2 );
		}
		else {
			System.out.println( "Invalid Operator!" );
		}
	}
}

File: php_Course/www/betterCalculator.php
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>

    <form action="betterCalculator.php" method="post">
      First Number: <input type="number" step="0.001" name="num1"> <br>
      Operator: <input type="text" name="op"> <br>
      Second Number: <input type="number" step="0.001" name="num2"> <br>
      <input type="submit">
    </form>

    <?php
      $num1 = $_POST["num1"];
      $num2 = $_POST["num2"];
      $op = $_POST["op"];

      function betterCalculator($num1, $num2, $op){
        if($op == "+"){
          echo $num1 + $num2;
        } elseif($op == "-"){
          echo $num1 - $num2;
        } elseif($op == "*"){
          echo $num1 * $num2;
        } elseif ($op == "/"){
          echo $num1 / $num2;
        } else {
          echo "Invalid operator.";
        }
      }

      betterCalculator($num1, $num2, $op);
    ?>

  </body>
</html>
